apps/evm/lib/evm/refunds/sstore.ex

```diff
@@ -0,0 +1,73 @@
+defmodule EVM.Refunds.Sstore do
+  alias EVM.{ExecEnv, Configuration}
+
+  # Refund given (added into refund counter) when the storage value is set to zero from non-zero.
+  @storage_refund 15_000
+
+  @spec refund({integer(), integer()}, ExecEnv.t()) :: integer()
+  def refund({key, new_value}, exec_env) do
+    if Configuration.eip1283_sstore_gas_cost_changed?(exec_env.config) do
+      eip1283_sstore_refund([key, new_value], exec_env)
+    else
+      case ExecEnv.get_storage(exec_env, key) do
```

Comment:
Love that we have extracted this into its own module. What do you think of extracting this `case` statement into a private function in this module so you have the same level of abstraction for the `refund` function?